clang/test/Sema/bitfield-preferred-type-sizing.c

```diff
@@ -0,0 +1,108 @@
+// RUN: %clang_cc1 %s -triple=x86_64-apple-darwin10 -fsyntax-only -verify -std=c11 -Wno-unused-value -Wno-unused-but-set-variable -Wbitfield-width -Wbitfield-enum-conversion
+
+enum A {
+  A_a,
+  A_b,
+  A_c,
+  A_d
+};
+
+struct S {
+  enum A a1 : 1; // #1
```

Comment:
```suggestion
  enum A a1 : 1; // #a1_decl
```

Reply:
Yeah, I just discovered while working on the C++ version of the test that bookmarks aren't required to be numbers :D